src/PhpSpreadsheet/Writer/Pdf/MPDF.php

```diff
@@ -73,7 +73,7 @@ public function save($pFilename)
         }
 
         //  Create PDF
```

Comment:
Doesn't work without this config.
        $config = ['tempDir' => 'tmp'];
        $pdf = new \Mpdf\Mpdf($config);